scala/private/phases/phase_compile.bzl

```diff
@@ -106,7 +106,7 @@ def _phase_compile_default(ctx, p, _args = struct()):
         ctx,
         p,
         _args.srcjars if hasattr(_args, "srcjars") else depset(),
-        _args.buildijar if hasattr(_args, "buildijar") else True,
+        _args.buildijar if hasattr(_args, "buildijar") else False,
```

Comment:
this forces bazel to use full jar, otherwise compiling does not find classes. i think this may be related to how scala3 uses tasty to ensure strong type safety, but not sure why it needs the .class implementations... (tasty does exist on the ijar)